Src/FluentAssertions/Streams/BufferedStreamAssertions.cs

```diff
@@ -27,6 +27,8 @@ public BufferedStreamAssertions(BufferedStream stream)
         {
         }
 
+        protected override string Identifier => "bufferedstream";
```

Comment:
I just had an extra look at how we do this elsewhere and as `Identifier` is included in a sentence it should be
```suggestion
        protected override string Identifier => "buffered stream";
```

Reply:
Changed this.